src/components/BoxApproval/Box.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { HStack, VStack, Button, Text, Spacer } from '@chakra-ui/react';
+import './BoxApproval.css';
+
+function Box() {
+  return (
+    <div className="box">
+      <HStack className="boxHeader">
+        <VStack spacing={0} align="left" className="boxHeaderInfo">
+          <Text fontSize="6xl">Box #</Text>
+          <Text>date</Text>
+        </VStack>
+        <Spacer />
+        <Button size="xs" className="approveButton">
```

Comment:
Could we change the "No" and "Yes" to X and checkmark icons as shown in the Figma? Y'all can use the Chakra icons for now: https://chakra-ui.com/docs/media-and-icons/icon